registry/app.py

```diff
@@ -57,6 +58,18 @@ def translate(input, rodne_cislo):
         else:
             return input
 
+    @app.template_filter("capitalize")
+    def capitalize(input):
+        output = None
+        for match in re.finditer(r"\w+", input):
```

Comment:
Teda pochopit, proč se používá toto a ne _split_ mi dalo docela zabrat. Ale je to tak moudré. Vtipné je, že to chytí i jednopísmenná slova a pro ta celá ta logika níže neudělá vůbec nic. To jen pro kontrast s tou snahou ušetřit si vytváření nového seznamu/řetězce. 😃 A že těch jednopísmenných slov v názvech ulic a měst máme!